asabiris/handlers/kafkahandler.py

```diff
@@ -22,10 +22,12 @@
 def check_config(config, section, parameter):
 	try:
 		value = config.get(section, parameter)
+		if not value:  # Checks for empty string or None implicitly
+			raise ValueError("Configuration parameter '{}' is empty in section '{}'".format(parameter, section))
 		return value
-	except configparser.NoOptionError:
-		L.error("Configuration parameter '{}' is missing in section '{}'.".format(parameter, section))
-		exit()
+	except (configparser.NoOptionError, ValueError) as e:
+		L.warning("Configuration issue: {}".format(e))
+		return None  # Returns None on
```

Comment:
I am sorry, but I don't understand that comment here = I already see it returns None from code, but "on" doesn't clear that case for me... Maybe it is ok and I am just not familiar with the code :)